ray-operator/controllers/ray/common/pod_test.go

```diff
@@ -1272,7 +1272,7 @@ func TestGenerateRayStartCommand(t *testing.T) {
 				NeuronCoreContainerResourceName: NeuronCoreRayResourceName,
 				"cloud-tpus.google.com/v3":      "tpu",
 			},
-			expected: `ray start --head  --num-gpus=1  --resources='{"tpu":8}' `,
+			expected: `ray start --head  --num-gpus=1  --resources='{"neuron_cores":4}' `,
```

Comment:
Couldn't cloud-tpus come before neuron?

Reply:
Sorting is based on the container resourceName, for neuron it is `aws.amazon.com/neuroncore`, so it would come first.

Reply:
ah gotcha, thanks